pages/agent/v3/cli_annotate.md

```diff
@@ -38,6 +38,27 @@ steps:
 
 <%= image "annotations-styles.png", alt: "Screenshot of available annotation styles" %>
 
+## Annotation ordering
+
+Annotations are ordered by priority (highest to lowest) and then by most recently created. Priority can be set with the `--priority` option when creating the annotation with `buildkite-agent annotate`. Priority ranges from `1` to `10`, with `10` being the highest priority and `1` being the lowest. If priority is not set, the default priority is `3`. If two annotations have the same priority, the one created most recently will be displayed first.
+
+This is an example pipeline showcasing using priority to order annotations:
```

Comment:
optional rephrase
`This is an example pipeline that showcases how to use priority to order annotations.`

Reply:
```suggestion
This is an example pipeline that showcases how to use priority to order annotations:
```